packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -228,7 +228,6 @@ const Select = <
       if (typeof selected === "string") {
         return undefined;
       }
-
```

Comment:
Should we put this newline back in to differentiate the comment from the preceding code?